packages/rest/src/lib/errors/HTTPError.ts

```diff
@@ -1,5 +1,5 @@
-import type { RequestBody } from './DiscordAPIError';
 import type { InternalRequest } from '../RequestManager';
+import type { RequestBody } from './DiscordAPIError';
```

Comment:
Those 2 needs a file extension no?

```suggestion
import type { InternalRequest } from '../RequestManager.js';
import type { RequestBody } from './DiscordAPIError.js';
```